elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -377,5 +400,22 @@ $pfe-cta--Color--fallback: #003366;
     --pfe-cta--Color--hover:            #{pfe-color(ui-complement)};
 }
 
+/// ===========================================================================
+/// WIND CTA VARIANTS
+/// ===========================================================================
 
-
+:host([pfe-priority="secondary"][pfe-variant="wind"]) {
+  --pfe-cta--BackgroundColor:           transparent;
+  --pfe-cta--BorderColor:               #{pfe-color(surface--border)};
+  --pfe-cta--Color:                     var(--pfe-broadcasted--color--ui-link, #{pfe-color(ui-link)});
+  --pfe-cta--FontWeight:                #{pfe-var(font-weight--normal)};
+
+  --pfe-cta--BackgroundColor--hover:    #{pfe-color(surface--border--lightest)};
+  --pfe-cta--BorderColor--hover:        #{pfe-color(surface--border)};
+  --pfe-cta--Color--hover:              #{pfe-color(ui-link--hover)};
+
+  --pfe-cta--Color--focus:               #{pfe-color(ui-link--hover)};
+  --pfe-cta--BorderColor--focus:         #{pfe-color(surface--border)};
+  --pfe-cta__inner--BorderColor--focus:  #{pfe-local(BorderColor--focus)};
```

Comment:
I think this inner border color value needs to be different, shouldn't it reference an inner border color focus?

Reply:
It's using the same color as the external border does on focus.  It looks good on render.

Reply:
Sorry the thing i was trying to troubleshoot was the inner-border on dark , which seems to be nearly the same color as the background of button
![image](https://user-images.githubusercontent.com/456863/67315285-3cb13000-f4d4-11e9-9e9a-2c28ab7ab403.png)

Reply:
I mean as far as I can tell there are no separate styles for `on="dark"`.

Reply:
I actually don't see any `on="dark"` styles anywhere in the CTA sass

Reply:
right, I meant conceptually, the broadcast colors come through on dark backgrounds. Either way, its probably not that big of a deal.